gr-zeromq/lib/base_impl.cc

```diff
@@ -31,8 +31,8 @@
 namespace gr {
   namespace zeromq {
 
-    base_impl::base_impl(int type, size_t itemsize, size_t vlen, int timeout, bool pass_tags)
-      : d_vsize(itemsize * vlen), d_timeout(timeout), d_pass_tags(pass_tags)
+    base_impl::base_impl(size_t type, size_t itemsize, size_t vlen, int timeout, bool pass_tags)
```

Comment:
The 'type' argument is of type 'int' in the zmq::socket_t constructor, and the current code reflects that, but you've changed it to size_t everywhere.  Why is that?